### PR TITLE
[fix](resource)Add s3 checker for alter resource

### PR DESCRIPTION
**Pull request by pengxiangyu**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by pengxiangyu** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.97 seconds
 stream load tsv:          473 seconds loaded 74807831229 Bytes, about 150 MB/s
 stream load json:         40 seconds loaded 2358488459 Bytes, about 56 MB/s
 stream load orc:          68 seconds loaded 1101869774 Bytes, about 15 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230306132731_clickbench_pr_109395.html

File: fe/fe-core/src/main/java/org/apache/doris/catalog/S3Resource.java [hunk 1]
```diff
@@ -201,6 +201,33 @@ public void modifyProperties(Map<String, String> properties) throws DdlException
                 throw new DdlException("current not support modify property : " + any.get());
             }
         }
+
+        boolean needCheck = !this.properties.containsKey(S3_VALIDITY_CHECK)
+                || Boolean.parseBoolean(this.properties.get(S3_VALIDITY_CHECK));
+        if (properties.containsKey(S3_VALIDITY_CHECK)) {
+            needCheck = Boolean.parseBoolean(properties.get(S3_VALIDITY_CHECK));
+        }
```
Comment (adonis0147) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/S3Resource.java:209]:
```suggestion
        boolean needCheck = Boolean.parseBoolean(this.properties.getOrDefault(S3_VALIDITY_CHECK, "false"));
```

**Review by adonis0147 — APPROVED** [review]
LGTM

**Pull request merged**